ADAL/src/telemetry/ADTelemetry.m

```diff
@@ -147,7 +136,7 @@ - (void)stopEvent:(NSString*)requestId
 }
```

Comment:
1. dispatcherEventNow -> call receive instead
2. calculate the responseTime as well